src/lib.rs

```diff
@@ -322,11 +329,53 @@ impl Client {
         watch::Watch::new(self, query)
     }
 
+    /// Disables [`Row`] types validation against the database schema.
+    /// Validation is enabled by default.
+    ///
+    /// # Warning
+    ///
+    /// While disabled validation will result in increased performance
+    /// (between 1.1x and 3x, depending on the data),
+    /// this mode is intended to be used for testing purposes only,
```

Comment:
> used for testing purposes only

Why? No, it's supposed to be used in code that is covered by straightforward integration tests.